includes/class-ed-solr.php

```diff
@@ -176,6 +176,7 @@ private function define_public_hooks() {
 		$this->loader->add_action( 'wp_enqueue_scripts', $plugin_public, 'enqueue_scripts' );
 		$this->loader->add_action( 'publish_post', $plugin_public, 'store_post' );
 		$this->loader->add_action( 'publish_page', $plugin_public, 'store_post' );
+		$this->loader->add_action( 'delete_post', $plugin_public, 'delete_post' );
 	}
```

Comment:
If you bin a post it is not deleted from solr. To handle this add the following line below delete_post

$this->loader->add_action( 'trash_post', $plugin_public, 'delete_post' );